app/javascript/react/components/CommentForm.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { browserHistory } from 'react-router';
+
+class CommentForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      body: "",
+      errors: []
+    }
+    this.handleFormChange = this.handleFormChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.clearState = this.clearState.bind(this)
+  };
+
+  handleFormChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  postComment(payload) {
+    fetch(`/api/v1/beers/${this.props.beerId}/reviews/${this.props.reviewId}/comments`, {
```

Comment:
Ideally you shouldn't nest resources 3-deep. It would be better to set up your routes so that this is actually located at `/reviews/${this.props.reviewId}/comments`